addons/medical_ui/functions/fnc_handleEffects.sqf

```diff
@@ -0,0 +1,31 @@
+/*
+ * Author: BaerMitUmlaut
+ * Handles any audible and visual effects of medical.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ */
+#include "script_component.hpp"
+
+private _disableAll = (!isNull curatorCamera) || {!alive ACE_player};
+if (_disableAll) exitWith {
+    [false, false] call FUNC(effectUnconscious);
+    [false]        call FUNC(effectPain);
+    [false]        call FUNC(effectBloodVolume);
+};
+
+// - Current state info -------------------------------------------------------
+private _bleedingStrength = [ACE_player] call EFUNC(medical,getBloodloss);
+private _bleeding         = _bleedingStrength > 0;
+private _bloodVolume      = ACE_player getVariable [QEGVAR(medical,bloodVolume), DEFAULT_BLOOD_VOLUME];
+private _unconscious      = ACE_player getVariable [QEGVAR(medical,isUnconscious), false];
+private _heartRate        = ACE_player getVariable [QEGVAR(medical,heartRate), 70];
```

Comment:
The default heart rate is 80, I guess we should also put it into `script_macros_medical.hpp`.